backend/src/main/java/botobo/core/infrastructure/s3/FakeS3Uploader.java

```diff
@@ -0,0 +1,24 @@
+package botobo.core.infrastructure.s3;
+
+import botobo.core.domain.user.User;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+
+@Profile({"local", "test"})
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class FakeS3Uploader implements FileUploader {
+    @Override
+    public String upload(MultipartFile multipartFile, User user) {
+        return user.getProfileUrl();
```

Comment:
```원래는 resources/images에 저장하여, 해당 파일을 file 프로토콜을 이용해 전달할 생각이었으나 브라우저 보안 상 안된다고 하네요 ㅠ_ㅠ 그래서 일단은 동작만 하게 냅뒀는데, 혹시 다른 방법 생각나시는 게 있으시다면 언제든 코멘트 남겨주세요!```

파일 프로토콜을 이용해 전달한 다는 게 어떤 건지 잘 모르겠네요😓
제가 상황을 이해한 바로는 resources/static 경로에 이미지를 저장해두고 url을 return 해주면 될 것 같습니다.
```suggestion
        // multipart 파일을 file로 변환 후 저장하는 로직
        return "http://localhost:8080/image.png";
```

개인적인 의견으로는 로컬 환경에서는 dev나 prod 처럼 동작하지 않아도 상관없다고 생각해요.
default-image 하나 두고 쓰는 것도 괜찮아 보입니다.
```suggestion
        return "http://localhost:8080/default-image.png";
```

Reply:
오 static에 담으면 저렇게 반환되나요??!!?!!! 디폴트이미지 하나두고 반환하는거 좋은 것같아요 !! file:// 쓰니깐 ㅇ보안이슈로 안되더라구여 ㅠㅠ

Reply:
테스트 환경에서 LocalStack을 활용한 테스트가 가능하면 로컬에서 테스트 하기보다는 테스트 코드만 실행시키면 되니 중간곰이 말한 것처럼 디폴트 이미지를 반환하거나 지금 코드 그대로 두어도 괜찮다고 생각해요!!
다만 현재 그게 힘든 상황이다보니..ㅠㅠ